src/refcount.rs

```diff
@@ -79,6 +82,10 @@ impl RefCounter for Strong {
     fn into_either(self) -> Either {
         Either::Strong(self)
     }
+
+    fn as_ptr(&self) -> *const AtomicBool {
```

Comment:
Not sure about having this in the RefCounter trait, this would make it public API, no?

Reply:
It's `#[doc(hidden)]` in the trait declaration of this already-for-internal-use trait.
Seemed less messy this way.
If we want people to implement more kinds of `RefCounter` themselves 😬 this could be a separate trait, so they could choose if their new `RefCounter` will participate in pointer shenanigans.

Reply:
Fair enough, it would have to be a pub trait anyway.